doc/source/ray-metrics.rst

```diff
@@ -0,0 +1,131 @@
+Monitoring Ray
+==============
+To help monitoring Ray applications, Ray
+
+- Collects Ray's pre-selected system level metrics.
+- Exposes metrics through Prometheus endpoints.
+- Supports application-level metrics collection APIs.
+
+.. note::
+
+    It is currently an experimental feature and under active development. APIs are subject to change.
+
+Getting Started (Single Node)
+-----------------------------
+You can access Ray's metrics through Prometheus endpoints.
+
+First, let's learn how to access metrics when you use `ray.init()`.
+
+.. code-block:: python
+
+    import ray
+    ray.init(_metrics_export_port=8080)
+    import time
+    time.sleep(100)
+
+Now you can access Prometheus endpoint at the address `http://localhost:8080`. Note that if you don't specify `_metrics_export_port`, 
+the port is randomly assigned. You can check the port number when you run `ray.init`.
+
+.. code-block:: bash
+
+    {'node_ip_address': '192.168.1.82',
+    'raylet_ip_address': '192.168.1.82',
+    'redis_address': '192.168.1.82:6379',
+    'object_store_address': '/tmp/ray/session_2020-08-04_13-50-35_156166_20127/sockets/plasma_store',
+    'raylet_socket_name': '/tmp/ray/session_2020-08-04_13-50-35_156166_20127/sockets/raylet',
+    'webui_url': 'localhost:8265',
+    'session_dir': '/tmp/ray/session_2020-08-04_13-50-35_156166_20127',
+    'metrics_export_port': 63301} # <= You can access metrics through http://localhost:[metrics_export_port]
+
+Second, let's learn how to expose metrics when you use `ray start`.
+
+.. code-block:: bash
+
+    ray start --head --metrics-export-port=8080 # Assign metrics export port on a head node.
+
+
+Now, you can scrape Ray's metrics using Prometheus. 
+
+First, download Prometheus.
```

Comment:
You should say it's exported in prometheus format, can be scraped by anything but here we'll use prometheus for simplicity